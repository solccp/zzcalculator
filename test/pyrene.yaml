---
structure_id: 1
number_of_atoms: 16
double_bonds:
- [1, 6]
- [2, 3]
- [10, 9]
- [16, 15]
- [5, 4]
- [12, 13]
- [11, 7]
- [8, 14]
---
structure_id: 2
number_of_atoms: 16
double_bonds:
- [1, 6]
- [2, 3]
- [10, 9]
- [16, 15]
- [5, 4]
- [12, 11]
- [13, 14]
- [7, 8]
---
structure_id: 3
number_of_atoms: 16
double_bonds:
- [1, 6]
- [2, 3]
- [10, 9]
- [16, 15]
- [5, 4]
rings:
- [12, 11, 7, 8, 14, 13]
---
structure_id: 4
number_of_atoms: 16
double_bonds:
- [1, 6]
- [2, 3]
- [10, 9]
- [16, 11]
- [15, 5]
- [4, 7]
- [12, 13]
- [14, 8]
---
structure_id: 5
number_of_atoms: 16
double_bonds:
- [1, 6]
- [2, 3]
- [10, 9]
- [8, 14]
- [12, 13]
rings:
- [16, 11, 7, 4, 5, 15]
---
structure_id: 6
number_of_atoms: 16
double_bonds:
- [1, 2]
- [6, 5]
- [15, 16]
- [3, 10]
- [4, 7]
- [11, 12]
- [9, 8]
- [13, 14]
---
structure_id: 7
number_of_atoms: 16
double_bonds:
- [1, 2]
- [6, 5]
- [15, 16]
- [3, 4]
- [10, 9]
- [14, 13]
- [12, 11]
- [8, 7]
---
structure_id: 8
number_of_atoms: 16
double_bonds:
- [1, 2]
- [6, 5]
- [15, 16]
- [3, 4]
- [10, 9]
- [14, 8]
- [13, 12]
- [11, 7]
---
structure_id: 9
number_of_atoms: 16
double_bonds:
- [1, 2]
- [6, 5]
- [15, 16]
- [3, 4]
- [10, 9]
rings:
- [14, 8, 7, 11, 12, 13]
---
structure_id: 10
number_of_atoms: 16
double_bonds:
- [1, 2]
- [6, 5]
- [15, 16]
- [14, 13]
- [12, 11]
rings:
- [3, 4, 7, 8, 9, 10]
---
structure_id: 11
number_of_atoms: 16
double_bonds:
- [10, 9]
- [15, 16]
- [7, 11]
- [12, 13]
- [14, 8]
rings:
- [1, 2, 3, 4, 5, 6]
---
structure_id: 12
number_of_atoms: 16
double_bonds:
- [10, 9]
- [15, 16]
- [7, 8]
- [14, 13]
- [12, 11]
rings:
- [1, 2, 3, 4, 5, 6]
---
structure_id: 13
number_of_atoms: 16
double_bonds:
- [10, 9]
- [15, 16]
rings:
- [1, 2, 3, 4, 5, 6]
- [7, 8, 14, 13, 12, 11]
