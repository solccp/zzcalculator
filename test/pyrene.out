 6 + 6 x + 1 x^2
 total: 13
